components/playlist/browser/resources/components/playlistPlayer.tsx

```diff
@@ -35,7 +37,10 @@ export default function PlaylistPlayer ({
           name={item.name}
           cached={item.cached}
           thumbnailUrl={item.thumbnailPath.url}
-          onClick={() => {}}
+          onClick={() => postMessageToPlayer({
+            actionType: types.PLAYLIST_ITEM_SELECTED,
+            data: { ...item, mediaPath: { url: item.mediaPath?.url } }
```

Comment:
It there some reason we can't do 
```suggestion
            data: { ...item, mediaPath: item.mediaPath }
```
here? Do we want to make sure we only send the url bit?